collect_app/src/main/java/org/odk/collect/android/utilities/IconUtils.java

```diff
@@ -52,17 +52,18 @@ public static Bitmap getBitmap(Context context, int drawableId) {
         return bitmap;
     }
 
-    public static Drawable getSubmissionSummaryStatusIcon(Context context, String instanceStatus) {
+    public static int getSubmissionSummaryStatusIcon(String instanceStatus) {
```

Comment:
I think `IconUtils` is not a good place for this method so maybe we could move it?

Reply:
Yeah, good point. I'll work out somewhere else for it to live.